src/node/build/index.ts

```diff
@@ -669,29 +669,30 @@ export async function ssrBuild(
 
   return build({
     outDir: 'dist-ssr',
+    emitIndex: false,
+    emitAssets: true, // needed for manifest.json
+    cssCodeSplit: false,
+    preserveEntrySignatures: 'strict', // so generated files can be consumed by server
+    minify: false,
     ...options,
     rollupPluginVueOptions: {
-      ...rollupPluginVueOptions,
-      target: 'node'
+      target: 'node',
+      ...rollupPluginVueOptions
     },
     rollupInputOptions: {
-      ...rollupInputOptions,
       external: resolveExternal(
         rollupInputOptions && rollupInputOptions.external
-      )
+      ),
+      ...rollupInputOptions
```

Comment:
If you look closer, `external` is meant to override `rollupInputOptions.external`.